addons/binding/org.openhab.binding.systeminfo.test/src/test/java/org/openhab/binding/systeminfo/test/SysteminfoOSGiTest.java

```diff
@@ -0,0 +1,1064 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
```

Comment:
2018?

Reply:
It is also a big test, does it test a single class?

Reply:
The year has been updated.
It is big because it tests `SysteminfoHandler` which has 33 channels. :open_mouth: